devito/ops/compiler.py

```diff
@@ -0,0 +1,17 @@
+from devito.parameters import configuration
+
+
+__all__ = ['CompilerOPS']
+
+
+class CompilerOPS(configuration['compiler'].__class__):
+    def __init__(self, *args, **kwargs):
+        super(CompilerOPS, self).__init__(*args, **kwargs)
+
+    def jit_compile(self, soname, code, hcode):
+        super(CompilerOPS, self).jit_compile(soname, code)
+        target = str(self.get_jit_dir().joinpath(soname))
+        h_file = "%s.h" % (target)
+        kernel_file = open(h_file, "w")
+        for kernel in hcode:
```

Comment:
why is this a loop?

Reply:
It shouldn't be, was using hcode as a list before, but changed it. Pushing the changes!